test/unit/utils.js

```diff
@@ -1,5 +1,6 @@
 'use strict'
 
+require('colors')
```

Comment:
Actually, this is required in both now. this test file uses colors to verify that the response has colors too.